Backpack/Button/Classes/BPKButton.m

```diff
@@ -155,9 +157,16 @@ - (void)setup {
     self.heightConstraint = [self.heightAnchor constraintEqualToConstant:[self heightForSize:self.size]];
     // To use when just icon
     self.widthConstraint = [self.widthAnchor constraintEqualToConstant:[self heightForSize:self.size]];
-    self.stackLeadingConstraint = [self.contentStack.leadingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor constant:BPKSpacingBase];
-    self.stackTrailingConstraint = [self.contentStack.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor
-                                                                                    constant:-BPKSpacingBase];
+    self.stackLeadingFallbackConstraint = [self.contentStack.leadingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor
+                                                                                          constant:BPKSpacingBase];
+    self.stackLeadingFallbackConstraint.priority = UILayoutPriorityDefaultLow;
+    self.stackTrailingFallbackConstraint = [self.contentStack.trailingAnchor constraintEqualToAnchor:self.contentView.trailingAnchor
+                                                                                            constant:-BPKSpacingBase];
+    self.stackTrailingFallbackConstraint.priority = UILayoutPriorityDefaultLow;
+    self.stackLeadingConstraint = [self.contentStack.leadingAnchor constraintGreaterThanOrEqualToAnchor:self.contentView.leadingAnchor
+                                                                                               constant:BPKSpacingBase];
+    self.stackTrailingConstraint = [self.contentStack.trailingAnchor constraintLessThanOrEqualToAnchor:self.contentView.trailingAnchor
+                                                                                              constant:-BPKSpacingBase];
```

Comment:
Setting up a secondary constraint for the content so that if size grows more than the content itself, the content remains centered